grants/milestone_deliveries/season_2/package_manager/package-manager-milestone-1.md

```diff
@@ -0,0 +1,47 @@
+# Milestone Delivery :mailbox: - Milestone 1
+
+* **Application Document:** [Grant Application - Package Manager](https://radicle.community/t/grant-application-package-manager/3122)
+* **Milestone Number:** Milestone 1
+
+**Context**
+
+The scope of this first milestone was to assess the feasibility of using Radicle as a package repository, for serving library packages such as the ones used by [npm](https://npmjs.org). The npm/JavaScript/GitHub ecosystem is very popular, and the research was conducted within that context.
+
+**Deliverables**
+
+The [research notes](./milestone1-part1-details.md) are presented here as deliverables for the first milestone. Summary videos are included, in addition to written notes. It was determined that it **is** possible to build a JavaScript application without any dependence on GitHub or npm. JavaScript packages (dependencies) can be retrieved directly from Radicle Seed nodes (as git repositories). There is some nuance with using Radicle as a package repository, which the research notes expand on.
+
+| Number | Deliverable | Link | Notes |
+| ------------- | ------------- | ------------- |------------- |
+| 1. | Generate an npm-compatible package from source code obtained from Radicle | [Details](./milestone1-part1-details.md) | Research notes |
+| 2. | Verdaccio Integration | [Details](./milestone1-part2-details.md) | Research notes |
+| 3. | Package Signing & Security | [Details](https://radicle.community/t/discussion-package-signing-security/3157) | Discussion Thread |
+
```

Comment:
The original Milestone 1 had 4 parts ([link here](https://radicle.community/t/grant-application-package-manager/3122#milestone-1-npm-compatibility-12))

Can we add a quick note explaining why we dropped 1 of the milestones?

I vaguely remember us talking about this, but would be good to have a quick notes (1-2 sentences on that)